main.nf

```diff
@@ -527,7 +527,7 @@ process sample_correlation {
      
      memory { 16.GB * task.attempt }
      time { 2.h * task.attempt }
-     errorStrategy { task.exitStatus == 143 ? 'retry' : 'warning' }
+     errorStrategy { task.exitStatus == 143 ? 'retry' : 'finish' }
```

Comment:
Do you know what happens to these processes if one or more steps has failed previously? I guess we don't want them to run if anything is missing.